plugins/nodes/src/js/components/NodesGridDials.d.ts

```diff
@@ -0,0 +1,12 @@
+import { ClassicComponentClass } from "react";
+
+interface NodesGridDialsProps {
+  hosts: any[];
+  resourcesByFramework: object;
+  selectedResource: string;
+  serviceColors: { [index: string]: number };
+}
+
+const NodesGridDials: ClassicComponentClass<NodesGridDialsProps>;
```

Comment:
The error here is just typescript complaining because it requires `declare` for any top level non-interface declarations in .d.ts files. Error goes away with `declare const NodesGridDials: ClassicComponentClass<NodesGridDialsProps>;`